### ISSUE TITLE
"user is offline" makes bottom bar chat useless

### ISSUE THREAD

**Issue by lostamongtrees** [RolePlayGateway/roleplaygateway#284]
![image](https://user-images.githubusercontent.com/52435234/77810040-cdce9900-7068-11ea-92cb-e77d633a3519.png)

cannot see what the other user sent, leads to endless back and forth context-less pings.
at this point id say scrap bottom bar chat or scrap the offline indicator. thing is a nuisance for too long and today i have a few responses i cannot access due to it.
